app/src/main/java/swati4star/createpdf/activity/ImageEditor.java

```diff
@@ -95,7 +95,7 @@ void initValues() {
         mFilterUris = getIntent().getStringArrayListExtra(IMAGE_EDITOR_KEY);
         mDisplaySize = mFilterUris.size();
         mFilterItems = getFiltersList(this);
-        mBrushItems = getBrushItems();
+        mBrushItems = getInstance().getBrushItems();
```

Comment:
use `BrushUtils.getInstance` here. It'll be easier to read.

Reply:
@Swati4star made the changes